internal/provider/resources/webhooks.go

```diff
@@ -185,12 +193,18 @@ func (r *WebhookResource) Create(ctx context.Context, req resource.CreateRequest
 		return
 	}
 
+	template, diags := helpers.UnmarshalOptional(plan.Template)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
 	createReq := api.WebhookCreateRequest{
 		WebhookCore: api.WebhookCore{
 			Name:             plan.Name.ValueString(),
 			Description:      plan.Description.ValueString(),
 			Enabled:          plan.Enabled.ValueBool(),
-			Template:         plan.Template.ValueString(),
+			Template:         template,
```

Comment:
This is the important part. It matches how we send payload fields in other cases, such as [block.Data](https://github.com/PrefectHQ/terraform-provider-prefect/blob/e3e6ce4b0d2596eb832815c08e3c6d08e2937e14/internal/provider/resources/block.go#L248-L252).